util/logutil/log.go

```diff
@@ -143,6 +144,24 @@ func initGRPCLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, error) {
 	return l, prop, nil
 }
 
+// ReplaceLogger replace global logger instance with given log config.
+func ReplaceLogger(cfg *LogConfig) error {
```

Comment:
Do we need to replace slow query logger to enable slow log runtime modification either?